cloud/amazon/rds.py

```diff
@@ -292,9 +297,9 @@
      wait: yes
      wait_timeout: 600
      tags:
-         Name: pg1_test_name_tag 
+         Name: pg1_test_name_tag
   register: rds
-  
```

Comment:
Generally we try to avoid making whitespace changes to areas you aren't editing. If possible, can you set your editor to not automatically do this?